Classes/Domain/Repository/GlossaryEntryRepository.php

```diff
@@ -48,6 +48,26 @@ public function findByLanguages(array $languages): array
         return $query->execute()->toArray();
     }
 
+    /**
+     * @param string $text
+     * @return GlossaryEntry[]
+     */
+    public function findByTextAndLanguage(string $text, string $language): array
+    {
+        $query = $this->createQuery();
+        $queryBuilder = $query->getQueryBuilder();
+
+        $result = $queryBuilder
+            ->select('e.glossaryLanguage')
```

Comment:
Could be count() to simplify the process and return a boolean directly?

Reply:
I adjusted it to select with count. `count()` is not available in this querybuilder version.